app/src/main/java/ingsw/group1/findmyphone/activity/MainActivity.java

```diff
@@ -40,6 +44,7 @@
     private EditText txtPhoneNumber;
     private ImageButton sendAlarmRequestButton;
     private ImageButton sendLocationRequestButton;
+    private Toolbar mainToolbar;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'mainToolbar' could be replaced by a local variable.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=5065092)